
function testexc.an_exception() 
   error "An exception raised from Lua code"
end

function testexc.an_error()
   foobarbla()
end

